.github/workflows/build-image.yml

```diff
@@ -11,7 +11,36 @@ on:
       - .github/workflows/build-image.yml
       - Dockerfile
 jobs:
-  build-image:
+  build-image-amd64:
```

Comment:
I think it's smart to use build matrix.

see https://docs.github.com/en/actions/using-workflows/advanced-workflow-features#using-a-build-matrix

Reply:
My main problem is the logging for both builds being piped to the same log file... so it jumps backforward and forwards between builds.

If matrix solves THIS problem. I'm good.

Reply:
![image](https://user-images.githubusercontent.com/17537000/158328991-159631a0-85b2-4dda-930e-e21969dd334d.png)

Reply:
They can be placed in the same group, but will be displayed separately.